ex-03/readme.md

```diff
@@ -83,13 +89,89 @@ Snyk has a SAST (Static Application Security Testing) module, let's try it.
   trufflehog git file://../.. --no-update
   ```
 
+- Explore Trufflehog options
```

Comment:
Should we already above mention something about `verify` 🤔 ?

Reply:
Also, should we explain why we run with `--no-update` (why this option and not any others)?

Reply:
A part of the exercise is to run "trufflehog --help" - this is typically where we browse through various key options and mention the --no-update as well. We have multiple reasons for using this option in the course; one is to make sure everyone have the exactly same environment. The other one is that you run Th update out of the box you need to "su" - and root will be the owner of the new executable, not "codespace" - so the update will be a bit more complex. There may also be a third option on deliberately wanting to update and not go full auto on "all stuff". In essence; this could be a complex discussion which may not be for this course......